lms/djangoapps/edraak_university/views.py

```diff
@@ -180,7 +185,7 @@ class UniversityIDStaffView(ContextMixin, generic.FormView, generic.ListView):
 
     def get_success_url(self):
         return reverse('edraak_university_id_success', kwargs={
-            'course_id': self.kwargs['course_id'],
+            'course_id': self.kwargs['course_id'], 'is_student': False,
```

Comment:
Good point, well you can check the type of user inside the view that renders the html and accordingly do the context data logic.